runtime/moonbase/tests/integration_test.rs

```diff
@@ -1042,22 +1040,17 @@ fn is_nominator_via_precompile() {
 			let expected_true_result = Some(Ok(PrecompileOutput {
 				exit_status: ExitSucceed::Returned,
 				output: expected_bytes,
-				cost: 0,
+				cost: 0, //TODO moonbase runtime still uses DbWeight = ()
```

Comment:
what is this TODO?

Reply:
This is a reminder that we need to update https://github.com/PureStake/moonbeam/blob/master/runtime/moonbase/src/lib.rs#L186

I didn't do it here because it is a bigger change than precompiles alone. I'll open a PR for it.

Reply:
https://github.com/PureStake/moonbeam/pull/520